apps/token/src/routes/rewards/home/reward-info.tsx

```diff
@@ -19,6 +19,10 @@ interface RewardInfoProps {
   rewardAssetId: string;
 }
 
+// Note: For now the only reward type is Staking. We'll need this from the API
+// at a later date
+const DEFAULT_REWARD_TYPE = 'Staking';
```

Comment:
This needed to be reverted as rewardType field is not on mainnet. This must be redone once this release goes out. Related to #930